src/internal/nft-api.ts

```diff
@@ -622,6 +624,40 @@ export async function getSpamContracts(
   );
 }
 
+export async function reportSpam(
+  config: AlchemyConfig,
+  contractAddress: string,
+  srcMethod = 'reportSpam'
+): Promise<void> {
+  void requestHttpWithBackoff<ReportSpamParams, void>(
+    config,
+    AlchemyApiType.NFT,
+    'reportSpam',
+    srcMethod,
+    {
+      contractAddress
+    }
+  );
+}
+
+export async function isAirdropNft(
+  config: AlchemyConfig,
+  contractAddress: string,
+  tokenId: string,
+  srcMethod = 'isAirdropNFT'
```

Comment:
nit: rename to `Nft`

Reply:
doesn't this one need to be the actual endpoint? which is spelled [isAirdropNFT](https://docs.alchemy.com/reference/isairdropnft-v3)?

Reply:
oh i see this is the one that gets logged, not the restApiName